tslearn/preprocessing.py

```diff
@@ -112,8 +114,8 @@ class TimeSeriesScalerMinMax(TransformerMixin):
     """
     def __init__(self, value_range=(0., 1.), min=None, max=None):
         self.value_range = value_range
-        self.min_ = min
```

Comment:
Why changing that?

Seems like a usual way to name attributes in sklearn that we try to mimic in tslearn, eg:
https://github.com/scikit-learn/scikit-learn/blob/1495f6924/sklearn/preprocessing/data.py#L233

Reply:
I don't know by heart anymore, but I think it was some kind of fix to make sure the `check_estimator` tests passed (same for the mean-variance scaler). 

I'll re-check on this and see if the tests pass without this change as well!

Reply:
I think the reason for this is the following: all the objects/parameters passed by the user to the constructor have to be stored under the same name in the self object. As the name of the parameters here are min and max, these have to be the same in the self-object to be sklearn-compatible. An alternative is to refactor the parameter names to min_ and max_ but that is kind of weird, and they are going to be deprecated soon anyway (and replaced for value_range)

I will undo the change one more time just to double check this, but I think the check_estimator will fail.

Reply:
I double checked it, and it will raise this error:

```
AssertionError: Estimator TimeSeriesScalerMinMax should not set any attribute apart from parameters during init. Found attributes ['max_', 'min_'].
```